src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
```

Comment:
The method getRandomColor() creates a new Random object every time it is called. This can be inefficient. Consider creating a single Random instance as a class-level field to improve performance.